p2p/rating/peersRatingHandler.go

```diff
@@ -173,6 +176,9 @@ func (prh *peersRatingHandler) GetTopRatedPeersFromList(peers []core.PeerID, min
 	prh.mut.Lock()
 	defer prh.mut.Unlock()
 
+	peersTopRated := make([]core.PeerID, 0)
+	defer prh.displayPeersRating(&peersTopRated, minNumOfPeersExpected)
```

Comment:
do we need to provide the reference here? a slice is always represented by a pointer, anyway

Reply:
yes, list is empty without reference